components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.scss

```diff
@@ -22,6 +22,7 @@
 
   &:disabled {
     color: $chef-primary-dark;
+    cursor: default; // no pointer when there's only one element
```

Comment:
found an issue here - if i've selected all projects, now i can no longer re-open the dropdown to change my project selections.

Reply:
Oh really? I've tried that, I believe... let me try again.

Reply:
<img width="191" alt="Screenshot 2019-06-05 at 20 25 28" src="https://user-images.githubusercontent.com/870638/58980218-1cf0c800-87d0-11e9-8b72-8301223b8a02.png">

Having no issues opening the dropdown when I have all projects selected before (and applied that selection). Can you please elaborate?

Reply:
I also am unable to reproduce the problem. If it is, in fact, a real issue lurking on some boundary conditions, it would not be caused by any changes from this PR.

Reply:
Oops this was due to an old gateway build running on my machine. nevermind! 😅